packages/serverless/src/apollo/app/user.ts

```diff
@@ -152,9 +163,15 @@ export const typeDefsUser = gql`
     publicKey: String!
   }
 
+  input FindUser {
```

Comment:
Can you show the example of the query to be passed? I do not really understand how the query fits this input

Reply:
<img width="969" alt="image" src="https://github.com/user-attachments/assets/63e611e8-a43f-4d66-bade-7c5e11bc155c">

Reply:
![Uploading image.png…]()